crowbar_framework/app/models/deployer_service.rb

```diff
@@ -181,6 +162,27 @@ def transition(inst, name, state)
       roles << node.run_list_to_roles
       roles.flatten!
 
+      # Allocate required addresses
+      range = node.admin? ? "admin" : "host"
+      @logger.debug("Deployer transition: Allocate admin address for #{name}")
+      ns = NetworkService.new @logger
+      result = ns.allocate_ip("default", "admin", range, name)
+      @logger.error("Failed to allocate admin address for: #{node.name}: #{result[0]}") if result[0] != 200
+      if result[0] == 200
+        address = result[1]["address"]
+        boot_ip_hex  = sprintf("%08X",address.split(".").inject(0){ |acc,i|(acc << 8)+i.to_i })
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/SpaceAfterComma: Space missing after comma. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)<br>Style/SpaceAroundBlockParameters: Space after closing `|` missing.<br>Style/SpaceAroundOperators: Surrounding space missing for operator `+`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)<br>Style/SpaceBeforeBlockBraces: Space missing to the left of {.